README.md

```diff
@@ -48,12 +62,33 @@ import SwiftCurrent
 launchInto(Workflow(ExampleViewController.self))
 ```
 
+### Original:
+
 And just like that you're started!  To see something more practical and in-depth, check out the example app in the repo.  For a more in-depth starting guide, checkout out our [Getting Started](https://github.com/wwt/SwiftCurrent/wiki/getting-started) documentation.
 
+### Option #1:
```

Comment:
I like that this one offers up clean links to the guides and links back to the example app instructions, but it seems a bit much.